llvm/unittests/CodeGen/SelectionDAGPatternMatchTest.cpp

```diff
@@ -152,6 +152,24 @@ TEST_F(SelectionDAGPatternMatchTest, matchBinaryOp) {
       sd_match(SFAdd, m_ChainedBinOp(ISD::STRICT_FADD, m_SpecificVT(Float32VT),
                                      m_SpecificVT(Float32VT))));
 
+  EXPECT_TRUE(sd_match(And, m_AnyBinOp(ISD::AND, m_Value(), m_Value())));
```

Comment:
Have you ran this test? Because I'm a little surprised this test can be built and passed since the signature of `AnyBinaryOp_match::match` doesn't conform to what `sd_match` expects.
In case you don't know how to run a unit test, try `ninja check-llvm-unit`. Another way for people lacks patience like me is to explicitly build unit test via `ninja UnitTests` and run only the test in this file via
```
./unittests/CodeGen/CodeGenTests --gtest_filter='SelectionDAGPatternMatchTest.*'
```
(`--gtest_filter` comes from google test, which we use for unit testing)